blocks/graph.py

```diff
@@ -37,6 +37,7 @@ def _get_variables(self):
         self.applies = set()
         self.application_calls = set()
         self.updates = []
+        self.auxiliary_variables = set()
```

Comment:
On this topic: I was just looking at using selectors to e.g. return the L2 weight norms of a specific set of bricks. I feel that if we're not careful, we'll end up having very similar functionality between the brick selectors and the computation graph. Any ideas on how work should exactly be distributed between these two? Which one of the two should be in charge of collecting particular regularization variables from the application calls?